### DISCUSSION TITLE
How create buttons in user keybord,wich will be always there?

### DISCUSSION THREAD

**Discussion by AntonNosovich** [def-studio/telegraph#548] [Q&A]

**Comment by fabio-ivona** [comment]
hi please refer to the documentation: https://defstudio.github.io/telegraph/features/reply-keyboards

**Reply by tobya** [reply]
This link seems to be missing unfortunatley.

**Reply by fabio-ivona** [reply]
The documentation has moved to our new site:  https://docs.defstudio.it/telegraph/v1/features/reply-keyboards
